lldb/source/Plugins/ObjectFile/XCOFF/ObjectFileXCOFF.cpp

```diff
@@ -0,0 +1,243 @@
+//===-- ObjectFileXCOFF.cpp
+//-------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "ObjectFileXCOFF.h"
+
+#include <algorithm>
+#include <cassert>
+#include <string.h>
+#include <unordered_map>
+
+#include "lldb/Core/Module.h"
```

Comment:
Are all of these needed for the thing that the file is doing? (Please don't reduce them too aggressively -- that leads to fragile headers -- but I don't think you're using e.g. LZMA or PointerUnion at the moment)

Reply:
Filtered some of the headers. Thanks!